PhysicsTools/NanoAOD/plugins/TableOutputBranches.h

```diff
@@ -54,7 +54,9 @@ class TableOutputBranches {
     int idx = tab.columnIndex(pair.name);
     if (idx == -1)
       throw cms::Exception("LogicError", "Missing column in input for " + m_baseName + "_" + pair.name);
-    pair.branch->SetAddress(const_cast<T *>(&tab.columnData<T>(idx).front()));  // SetAddress should take a const * !
```

Comment:
`front()` returns a reference and should never be called if the container is empty.